src/MediaManagerBundle/Bridge/Chameleon/TableEditor/CmsMediaTreeTableEditor.php

```diff
@@ -22,20 +22,26 @@
 class CmsMediaTreeTableEditor extends TCMSTableEditorMediaTree
 {
     /**
-     * {@inheritdoc}
+     * {@inheritDoc}
      */
     protected function updateCache($treeNodeId)
     {
         $languageId = $this->getLanguageService()->getActiveEditLanguage()->id;
         $mediaTreeNode = $this->getMediaTreeNodeDataAccess()->getMediaTreeNode($treeNodeId, $languageId);
         if (null !== $mediaTreeNode) {
             /**
-             * @var $imageTableEditor CmsMediaTableEditor
+             * @var CmsMediaTableEditor $imageTableEditor
              */
             $imageTableEditor = TTools::GetTableEditorManager('cms_media')->oTableEditor;
+
             try {
                 $images = $this->getMediaItemDataAccess()->getMediaItemsInMediaTreeNode($mediaTreeNode, $languageId);
             } catch (DataAccessException $e) {
+                /**
+                 * @psalm-suppress InvalidArgument
+                 * @FIXME Incorrect constructor arguments to `ErrorException`. Should be the following:
+                 * throw new ErrorException($e->getMessage(), 0, 1, $e->getFile(), $e->getLine(), $e);
+                 */
                 throw new ErrorException($e->getMessage(), 0, $e);
```

Comment:
This exception construction is not valid. I don't know what behaviour this yields - so I did not change it for now.